test/selenium/src/main/java/ca/bc/gov/ecc/ccof/extentreport/ExtentManager.java

```diff
@@ -0,0 +1,21 @@
+package ca.bc.gov.ecc.ccof.extentreport;
+
+import com.aventstack.extentreports.ExtentReports;
+import com.aventstack.extentreports.reporter.ExtentSparkReporter;
+
+public class ExtentManager {
+	private ExtentManager() {
+	}
```

Comment:
Even though the constructor is private it should still be defined after any instance variables (i.e. extentReports).